cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -143,15 +144,18 @@ ifneq ($(ARCH),i686)
 CMD += -64
 endif
 
+# Make sure to use bash for pipefail setting
+SHELL=/bin/bash
```

Comment:
We use `pipefail` in other Makefiles as well and generally assume bash as shell in all of them. So this should go into a higher-level makefile, e.g. Makefile.sim